README.md

```diff
@@ -61,6 +61,12 @@ The major OpenYurt components consist of:
   reverse proxy to establish a secure network access between the cloud site control plane and the edge nodes 
   that are connected to the intranet.
 
+## Addons
```

Comment:
how about separate all components into cloud components and edge components, not add a Addons chapter?